src/Signer/Ecdsa/Asn1.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\JWT\Signer\Ecdsa;
+
+use FG\ASN1\ASNObject;
+use FG\ASN1\Universal\Integer;
+use FG\ASN1\Universal\Sequence;
+use InvalidArgumentException;
+use const STR_PAD_LEFT;
+use function assert;
+use function bin2hex;
+use function gmp_init;
+use function gmp_strval;
+use function hex2bin;
+use function mb_strlen;
+use function mb_substr;
+use function str_pad;
+
+final class Asn1 implements PointsManipulator
```

Comment:
This implementation doesn't appear to be tested

Reply:
It's being tested through the ecdsa test, which is not ideal (just like the OpenSSL)